src/components/LockButton.jsx

```diff
@@ -13,6 +13,7 @@ const LockButton = ({onClick, readOnly}) => {
       label={`Switch to ${ readOnly ? 'edit' : 'read-only' } mode`}
       onTouchTap={onClick}
       tooltip={toolTipText}
+      style={{zIndex: 5}}
```

Comment:
Might be against best-practice on this one.
http://www.material-ui.com/#/customization/styles